Trezor: displayaddress

Sjors opened the pull request:
Tested with Trezor T for mainnet, testnet using legacy, p2sh-wrapped and native Segwit addressees.

achow101 commented:
Please rebase onto master now that #66 has been merged.

Sjors commented:
@achow101 rebased

achow101 commented on hwilib/devices/trezor.py:
```diff
@@ -177,8 +177,13 @@ def sign_message(self, message, keypath):
-        raise NotImplementedError('The HardwareWalletClient base class does not '
-            'implement this method')
+      expanded_path = tools.parse_path(keypath)
+      output = self.client.get_address(
```
trezor 0.11.0 moved this to `trezorlib.btc.get_address`.

Sjors replied:
Ok, I'll rebase.

Sjors commented:
Rebased to support the new Trezor Python driver. Also added tests, though I haven't run those locally and I'm not sure if they would actually catch an error.

achow101 commented:
tACK 1e7ac4015d0163675d96057280fb1aa49dfdcebb

The test should at least check that the trezor api is being used correctly.
